frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/namespace-store-pages/multi-namespace-store.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Flex, FlexItem, Form, FormGroup, Stack, Title } from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { NamespaceStoreKind } from '../../../../types';
+import { NamespaceStoreList } from '../../../namespace-store/namespace-store-table';
+import { Action, State } from '../../state';
+import NamespaceStoreModal from '../../../namespace-store/namespace-store-modal';
+import { NamespaceStoreDropdown } from '../../../namespace-store/namespace-store-dropdown';
+
+export const MultiNamespaceStorePage: React.FC<MultiNamespaceStoreProps> = React.memo(
+  ({ state, dispatch, namespace }) => {
+    const { t } = useTranslation();
+    const openModal = () => NamespaceStoreModal({ namespace });
+    const [selectedCount, setSelectedCount] = React.useState(state.readNamespaceStore.length);
+    const onSelectNamespaceStoreTable = (selectedNamespaceStore: NamespaceStoreKind[]) => {
+      dispatch({ type: 'setReadNamespaceStore', value: [...selectedNamespaceStore] });
+      setSelectedCount(selectedNamespaceStore.length);
+    };
+
+    const onSelectNamespaceStoreDropdown = (selectedNamespaceStore: NamespaceStoreKind) => {
+      dispatch({ type: 'setWriteNamespaceStore', value: [selectedNamespaceStore] });
+    };
```

Comment:
nit: add spaces above and below
```suggestion

    const onSelectNamespaceStoreTable = (selectedNamespaceStore: NamespaceStoreKind[]) => {
      dispatch({ type: 'setReadNamespaceStore', value: [...selectedNamespaceStore] });
      setSelectedCount(selectedNamespaceStore.length);
    };

    const onSelectNamespaceStoreDropdown = (selectedNamespaceStore: NamespaceStoreKind) => {
      dispatch({ type: 'setWriteNamespaceStore', value: [selectedNamespaceStore] });
    };
    
```

Reply:
done